src/rpc/blockchain.h

```diff
@@ -8,6 +8,7 @@
 #include <consensus/amount.h>
 #include <core_io.h>
 #include <fs.h>
+#include <rpc/util.h>
```

Comment:
Addinng 400 LOC here doesn't seem to be an improvement relative to my last ACKed version.

Reply:
I'm going to revert it so. Addressed that thinking about keeping the code more organized but you're right about adding 400 LOC.